include/aspect/geometry_model/ellipsoidal_chunk.h

```diff
@@ -100,6 +100,11 @@ namespace aspect
           std::unique_ptr<Manifold<dim,3>>
           clone() const override;
 
+          /**
+           * A pointer to the topography model.
+           */
+          const InitialTopographyModel::Interface<dim> *topography;
```

Comment:
You made the same variable `public` in other classes too, right?

Reply:
Yes, except for merged_box and merged_chunk geometry models.